service/src/main/java/bio/terra/landingzone/stairway/common/model/TargetManagedResourceGroup.java

```diff
@@ -0,0 +1,3 @@
+package bio.terra.landingzone.stairway.common.model;
+
+public record TargetManagedResourceGroup(String name, String region) {}
```

Comment:
Can you add documentation? I don't know what this is/how it is intended to be used.